project-api-tests/src/test/java/org/ballerina/projectapi/CentralTestUtils.java

```diff
@@ -72,9 +68,9 @@ private CentralTestUtils() {
     static final String JAVA11_PLATFORM = "java11";
     static final String BALLERINA_ARTIFACT_TYPE = "bala";
     static final String OUTPUT_NOT_CONTAINS_EXP_MSG = "build output does not contain expected message:";
-    static final String CANNOT_RESOLVE_MODULE_MSG = "cannot resolve module '";
-    static final String DEPENDENCY_PULLING_ERR_MSG = "error: dependency pulling contains errors";
     static final String PULLED_FROM_CENTRAL_MSG = " pulled from central successfully";
+    static final String UNDERSCORE_SEPARATOR = "_";
```

Comment:
IMO, our constants need to reflect the purpose of the constant. So, it's better to name the constants as `PACKAGE_NAME_SEPARATOR` and `PATH_SEPARATOR`.WDYT?

Reply:
changed.